### PR TITLE
Flux 0.83

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/reference/flux/stdlib/influxdb-tasks/lastsuccess.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Parameters
+
+### orTime
+The default time value returned if the task has never successfully run.
+
+_**Data type:** Time_
+
+## Examples
+
+##### Query data since that last successful task run
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/influxdb-tasks/lastsuccess.md:29]: the last

**Review by kelseiv — APPROVED** [review]

File: content/influxdb/v2.0/reference/release-notes/flux.md [hunk 2]
```diff
@@ -9,11 +9,62 @@ menu:
...
+- Add Rust JSON tests.
+- CSV no longer deadlocks when next transformation does not consume table.
+
+---
+
+## v0.82.2 [2020-08-25]
+
+### Features
+- Add [`tasks.lastSuccess` function](/influxdb/v2.0/reference/flux/stdlib/influxdb-tasks/lastsuccess/)
+  to retrieve the time of the last successful run of an InfluxDB task.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.0/reference/release-notes/flux.md:47]: The feature was actually added in v0.82.2, but they had to add a semantic commit message for it to show up in the changelog. The "message-only" commit was part of 0.83. So if users do build from 0.82.2, they'll still have access to this function.

File: content/influxdb/v2.0/reference/release-notes/flux.md [hunk 3]
```diff
@@ -9,11 +9,62 @@ menu:
...
+## v0.82.1 [2020-08-25]
+- _Internal code cleanup._
+
+---
+
+## v0.82.0 [2020-08-24]
+
+### Features
+- Add the [`profiler` package](/influxdb/v2.0/reference/flux/stdlib/profiler/).
+- Add a documentation URL field to Flux errors.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.0/reference/release-notes/flux.md:60]: That's behind the scenes. This field is for the Flux team only. It allows them to add doc links to error messages.

File: content/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/integral.md [hunk 4]
```diff
@@ -20,23 +20,46 @@ _**Function type:** Aggregate_
...
+_**Data type:** String_
+
+### interpolate
+Type of interpolation to use.
+Defaults to `""`.
+
+Use one of the following interpolation options:
+
+- linear
+
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/integral.md:57]: Well, there are currently two options. "linear" or an empty string. An empty string means no interpolation. I've added this to the list. Other interpolation methods are in the works.

**Pull request merged**